src/main/java/edu/matc/covidPulse/controller/AllService.java

```diff
@@ -1,19 +1,58 @@
 package edu.matc.covidPulse.controller;
```

Comment:
As you said in slack, you've scrapped this controller. Can we remove it, then?

Reply:
Yeah, it would be safe to delete it.